source/main/windows/main.js

```diff
@@ -2,13 +2,17 @@ import path from 'path';
 import { app, BrowserWindow, ipcMain, Menu } from 'electron';
 import environment from '../../common/environment';
 import ipcApi from '../ipc-api';
+import { runtimeFolderPath } from '../config';
 
 export const createMainWindow = () => {
+  const iconPath = runtimeFolderPath + '/icon.png';
+
   // Construct new BrowserWindow
   const window = new BrowserWindow({
     show: false,
     width: 1150,
-    height: 870
+    height: 870,
+    icon: iconPath,
```

Comment:
@cleverca22 what is the purpose of this icon? It is fine to add it I guess, but I can not locate the `icon.png` on `macOS` install.
Also we have AboutWindow - not sure if we need this icon there too?

Reply:
on linux, this is the only way to put an icon on the window
i can add some if statements to make it do nothing on other platforms

Reply:
We have checked in macOS and Windows and it looks as this icon import has no affect.
It could be a Linux specific icon so I don't think any "ifs" are necessary.